src/Maestro/MaestroApplication/ApplicationPackageRoot/ApplicationManifest.xml

```diff
@@ -132,6 +132,6 @@
     <DefaultRunAsPolicy UserRef="MaestroUser" />
   </Policies>
   <Certificates>
-    <EndpointCertificate X509FindType="FindBySubjectName" X509FindValue="[WebCertName]" Name="WebSslCert" />
+    <EndpointCertificate X509FindValue="[WebCertName]" Name="WebSslCert" />
```

Comment:
I see VS is still removing this. We should probably put it back.

Reply:
Damn it. :-|